### ISSUE TITLE
Point2 with missing

### ISSUE THREAD

**Issue by yakir12** [MakieOrg/Makie.jl#1389]
While `missing` in the `x` or `y` vectors of numbers works:
```julia
using GLMakie
x = [1, missing, 3, 4]
y = 1:4
scatter(x, y) # works
```
Or a `missing` in the `xy` vector of `Point2` works:
```julia
xy = [i == 2 ? missing : Point2(i, i) for i in 1:4]
scatter(xy) # works
```
a `missing` value inside one of the points in a vector of `Point2` doesn't:
```julia
x = [1, missing, 3, 4]
y = 1:4
xy = Point2.(x, y)
scatter(xy) # doesn't work
```
nor two `missing` values:
```julia
xy = [i == 2 ? Point2(missing, missing) : Point2(i, i) for i in 1:4]
scatter(xy) # doesn't work
```

This isn't all that surprising, but I'll leave it here just in case.

**Comment by ffreyer** [comment]
The latter two cases produce `Vector{Point2}`, i.e. no `Union{Missing, ...}`, so we would have to catch them with a `convert_arguments(::PointBased, ps::Vector{VecTypes{N}})` method. We probably don't want to do something like that since it would trigger on all kinds of things and may mess with dim converts?

**Comment by yakir12** [comment]
I agree. Better close this and if someone ever wonders they will hopefully find this issue and understand the reasoning.
